_opensearch/remote.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: default
+title: Remote-backed storage
+nav_order: 19
+---
+
+# Remote-backed storage
+
+Remote-backed storage offers OpenSearch users a new way to protect against data loss by automatically creating backups of all index transactions and sending them to a supported cloud storage service. In order to expose this feature, segment replication must also be enabled. See [Segment replication]({{site.url}}{{site.baseurl}}/opensearch/segment-replication/) for additional information.
```

Comment:
protect against = avoid?

Reply:
Both work but I feel like "protect against" is more effective. I'm thinking about all the marketing and advertising I see for products that offer some kind of protection (like "identity theft protection" usually, instead of "avoiding identify theft," I think).  I also am concerned that to say "avoid data loss" imputes some kind of guarantee, whereas protecting against data loss reminds the reader that data loss is an ever-present concern and multiple, redundant protections should be in place to mitigate the risk.

Reply:
Agreed. It just feels like "protect" needs a reference to whom to protect... Maybe "protect data against loss"?

Reply:
"data loss" is what you are protecting against.